### ISSUE TITLE
My Grad Cam is giving opposite results.

### ISSUE THREAD

**Issue by snehitvaddi** [rigvedrs/YOLO-V8-CAM#13]
I'm working on a project where I trained a YOLOv8 model to find E.coli bacteria droplets on leaves. After using Grad-CAM on the YOLO model, which was 100% accurate, I noticed something unexpected. The areas without droplets are marked in red, and the areas with droplets are in blue, which means they're considered less important.

Does this mean the model is actually working well and just the color scheme is flipped, or did I make a mistake somewhere?

Here are the Grad-CAM results and the original image displayed side by side for comparison:

<p float="left">
  <img src="https://github.com/rigvedrs/YOLO-V8-CAM/assets/35267863/f87b775b-6b13-4bf3-91fa-7483500e5fbb" width="400" />
  <img src="https://github.com/rigvedrs/YOLO-V8-CAM/assets/35267863/53515f84-50d7-4edc-b22d-f34d6382840b" width="400" /> 
</p>

**Comment by rigvedrs** [comment]
Yes, it could mostly be related to a switch in RGB channels. OpenCV has a common issue of considering a different order of channels than other packages, so if you're using two different packages for the same image, considering this might help.

**Comment by snehitvaddi** [comment]
Umm...that sounds like a fair reason but I am pretty sure that I haven't used any other image processing module other than OpenCV.

**Comment by TempestMars** [comment]
Hello, I get a very similar result. Have you figured out what is the cause? Thanks!
